libraries/CZ80Cpu/CLadybugBaseGame.cpp

```diff
@@ -0,0 +1,149 @@
+// OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+// COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+// EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+// HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
+// TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+// EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+#include "CLadybugBaseGame.h"
+#include "CZ80ACpu.h"
+#include <DFR_Key.h>
+
+//
+// Probe Head GND:
+//   Z80 GND Pin 29
+//
+// Watchdog Disable:
+//   No watchdog
+//
+// Board Designations:
+//   This is the CPU/main board base.
+//
+// Z80 Compatibility Notes
+//   There is no VBLANK interrupt, it is read from IN1
+//   Left coin generates INT, right coin generates IRQ
+
+
+//
+// RAM region is the same for all games on this board set.
+//
+static const RAM_REGION s_ramRegion[] PROGMEM = { //                                            "012", "012345"
+                                                  {NO_BANK_SWITCH, 0x006000, 0x0063FF, 1, 0x0F, " F3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006000, 0x0063FF, 1, 0xF0, " H3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006400, 0x0067FF, 1, 0x0F, " J3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006400, 0x0067FF, 1, 0xF0, " K3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006800, 0x006BFF, 1, 0x0F, " L3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006800, 0x006BFF, 1, 0xF0, " M3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006C00, 0x006FFF, 1, 0x0F, " N3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x006C00, 0x006FFF, 1, 0xF0, " P3", "CpuRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x00D000, 0x00D3FF, 1, 0x0F, " K6", "VidRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x00D000, 0x00D3FF, 1, 0xF0, " J6", "VidRam"}, // 2114
+                                                  {NO_BANK_SWITCH, 0x00D400, 0x00D8FF, 1, 0x0F, " L6", "ColRam"}, // 2114
+                                                  {0}
+                                                }; // end of list
+
+//
+// RAM region is the same for all games on this board set.
+//
+static const RAM_REGION s_ramRegionByteOnly[] PROGMEM = { //                                                                                      "012", "012345"
```

Comment:
NIT: The comment with the character count columns don't line up, same with the write only region below.